src/pages/Search/SearchAdvancedFiltersPage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import FullPageNotFoundView from '@components/BlockingViews/FullPageNotFoundView';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import AdvancedSearchFilters from './AdvancedSearchFilters';
+
+function SearchAdvancedFiltersPage() {
```

Comment:
Hmm I wonder if we need to break this down into `AdvancedSearchFilters` and `SearchAdvancedFiltersPage`. Maybe we can consolidate them into a single `AdvancedSearchFiltersPage`?

Reply:
Sounds good, let's keep it separate then!